content/en/tutorial/01-vdom.md

```diff
@@ -140,8 +140,7 @@ _two_ types of Virtual DOM elements. The other type is a Component, which is
 a Virtual DOM element where the type is a function instead of a string like `p`.
 
 Components are the building blocks of Virtual DOM applications. For now, we'll
-create a very simple component by moving our JSX into a function, which will be
-rendered for us so we don't need to write that last `render()` line anymore:
+create a very simple component by moving our JSX into a function:
```

Comment:
I get what this was going for, but I think it will be quite confusing for new users (as #974 rightly brings up).